pages/About Data Triad.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div class="container">
```

Comment:
All tags need a closing tag. This is the reason for the linting errors. This is the same for the `<article>` tags.

```html
<div>
  ...
</div>